docs/backend/openai_api_completions.ipynb

```diff
@@ -223,6 +223,13 @@
     "## Structured decoding (JSON, Regex)\n",
```

Comment:
You can define a JSON schema or regular expression to constrain the model's output, which depends on the grammar backend.

SGlang has two backends: outlines (default) and Xgrammar. Xgrammar enhances JSON decoding performance but does not support regular expressions. To use Xgrammar, add --grammar-backend xgrammar when launching the server:

Reply:
Updated and verified the functionality with Xgrammar backend.